extensions/vscode/CHANGELOG.md

```diff
@@ -1,3 +1,183 @@
+# 0.2.4
+
+- chore(deps-dev): bump eslint from 8.49.0 to 8.50.0 in /extensions/vscode ([#1075](https://github.com/VeryGoodOpenSource/dart_frog/pull/1075))
```

Comment:
Imho, we should remove all the deps-dev, since the CHANGELOG is user-facing and these changes do not impact the end-user and are merely tooling updates to facilitate the extension maintaince.

Reply:
So two things.

1. That's literally the entire changelog 🙃
2. We have it in previous versions already as well.